src/hidimstat/empirical_thresholding.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+from numpy.linalg import norm
+from sklearn.model_selection import GridSearchCV
+from sklearn.svm import LinearSVR
+
+
+def empirical_thresholding(
+    X,
+    y,
+    linear_estimator=GridSearchCV(
+        LinearSVR(), param_grid={"C": np.logspace(-7, 1, 9)}, n_jobs=None
+    ),
```

Comment:
I was just wondering why you selected this default estimator. 
An alternative could be the classical `RidgeCV`.

Reply:
I used one of the previous codes.

Reply:
Do you want to replace LinearSVR by Ridgecv? @jpaillard

Reply:
Do we keep like this?

Reply:
I suggest yes.